ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -216,6 +232,53 @@ data Mempool m blk idx = Mempool {
     , zeroIdx        :: idx
     }
 
+-- | The result of attempting to add a transaction to the mempool.
+data MempoolAddTxResult blk
+  = TxAdded
+    -- ^ The transaction was added to the mempool.
+  | TxRejected !(MempoolAddTxError blk)
```

Comment:
Should these constructors be prefixed with `Mempool`?

Reply:
Hmm, for consistency, yeah.